GliaWidgets/Sources/ViewModel/Chat/ChatViewModel.swift

```diff
@@ -788,6 +817,65 @@ extension ChatViewModel {
     }
 }
 
+// MARK: Message sending retry
+
+extension ChatViewModel {
+    private func retryMessageSending(_ outgoingMessage: OutgoingMessage) {
```

Comment:
This should be covered with unit tests probably too.

Reply:
Unit and snapshot tests will be added in next PRs

Reply:
Does it make sense to add comment here that this is to be unit tested in the next PR, just to avoid forgetting to do so?

Reply:
Added TODO cc591a67627cbd478b6539abe5d46b4a5940c19d